### PR TITLE
Change menu item name to avoid confusion

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
I got confused with the menu item title so I'm changing the text to explicitly say it's for logging in instead of running. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [x] Chore: cleanup/renaming, etc  
- [ ] RFC  

## Acceptance
- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
         >
             <add-to-group group-id="EditorPopupMenu" anchor="first"/>
         </action>
-        <group id="WatermelonContextGroup" popup="true" text="Watermelon Context">
+        <group id="WatermelonContextGroup" popup="true" text="Login to Watermelon">
```
Comment (EstebanDalelR) [inline src/main/resources/META-INF/plugin.xml:29]: Will this group ONLY be for login?

Comment (baristaGeek) [reply src/main/resources/META-INF/plugin.xml:29]: I see no other action so yes

Comment (EstebanDalelR) [reply src/main/resources/META-INF/plugin.xml:29]: So when we add the logout action we should rename it?

Comment (baristaGeek) [reply src/main/resources/META-INF/plugin.xml:29]: Yes. To something that's more clear than "Watermelon Context" such as "Login/Logout from Watermelon"

Comment (EstebanDalelR) [reply src/main/resources/META-INF/plugin.xml:29]: Should it then not be a group?

Comment (baristaGeek) [reply src/main/resources/META-INF/plugin.xml:29]: But we want to have a logout action soon, so why remove it?

**Review by EstebanDalelR — APPROVED** [review]

**Pull request merged**